torch/_torch_docs.py

```diff
@@ -8036,6 +8038,9 @@ def merge_dicts(*dicts):
 .. note:: When :attr:`compute_uv` = ``False``, backward cannot be performed since `U` and `V`
           from the forward pass is required for the backward operation.
 
+.. note:: With the complex-valued input the backward operation works correctly only
```

Comment:
Future work: this is too vague and referencing other documentation is never ideal